src/mockup-entry.ts

```diff
@@ -5,11 +5,14 @@ import * as languageMap from '@/mock-data/data/en_lang_data.json';
 import * as directionalities from '@/mock-data/data/en_lang_dir_data.json';
 import ImmediatelyInvokingEntityLoadedHookHandler from '@/mock-data/ImmediatelyInvokingEntityLoadedHookHandler';
 import MwWindow from '@/client/mediawiki/MwWindow';
+import getOrEnforceUrlParameter from './mock-data/getOrEnforceUrlParameter';
+
+const language = getOrEnforceUrlParameter( 'language', 'de' );
```

Comment:
The hard-wired "language" value could definitely come from a central place (see ` TermboxHandler`, `QueryValidator`) but let's wait for things to blow over in the SSR request department before investing more into this.